ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -113,7 +115,8 @@ export class Auth0AdapterService implements OnDestroy {
             }),
             ...(additionalParams && {
                 ...additionalParams
-            })
+            }),
+            language: this.language.getCurrentLanguage()
```

Comment:
@yufengwng, have we got the final decision about the parameter's name?

Reply:
I think I like just calling it `language`. I feel `selected_language` might be too specific (imagine if in the future a client automatically detects the language instead of having a user "select" it).